scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -154,7 +159,7 @@ def perform_migration(version):     # post-upgrade Data Migration (to be run as
         print("Updating application name in audit events table")
         updateApprovalGateAudit()
 
-        commit_transactions()
+        #commit_transactions()
```

Comment:
why this line is commented ?

Reply:
missed to remove commented while commit. now resolved